include/ccf/service/tables/service.h

```diff
@@ -12,14 +12,16 @@ namespace ccf
     OPENING = 1,
     OPEN = 2,
     WAITING_FOR_RECOVERY_SHARES = 3,
-    CLOSED = 4 // For now, unused
+    RECOVERING = 4,
+    CLOSED = 5 // For now, unused
```

Comment:
I think we should axe CLOSED, it's been there for years and there's still no credible plan to use it for anything. It can be added when there is.